### PR TITLE
Adds .gitattributes file for fewer merge conflicts

### PR DESCRIPTION
**Pull request by jmannfeld**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1 @@
+readme.md merge=union
```
Comment (styfle) [inline .gitattributes:1]: Lets put this in a separate PR

File: readme.md [hunk 2]
```diff
@@ -354,6 +354,12 @@ As in: OOPs I used Java!
 
 ---
 
+My girlfriend told me to take the spider out instead of killing it.
+
+We went and had some drinks. Cool guy. Wants to be a web developer.
+
+---
```
Comment (styfle) [inline readme.md:361]: The joke doesn't seem very developer focused 🤷

**Comment by wesbos** [comment]
I had no idea this was a thing! Amazing

**Review by styfle — APPROVED** [review]
Thanks!
